app/models/manageiq/providers/openstack/cloud_manager/cloud_volume.rb

```diff
@@ -51,6 +56,27 @@ def raw_delete_volume
     raise MiqException::MiqVolumeDeleteError, e.to_s, e.backtrace
   end
 
+  def cloud_volume_backup_create(options)
```

Comment:
Do you think we can remove cloud_volume from that method name and the feature name?

Other than that the supports thing is exactly what I meant :)